rqmonitor/utils.py

```diff
@@ -91,7 +91,10 @@ def attach_rq_worker_prefix(worker_id):
     for worker_instance in [Worker.find_by_key(attach_rq_worker_prefix(worker_id))
                             for worker_id in worker_ids]:
         requested_hostname = worker_instance.hostname
-        requested_hostname = requested_hostname.decode('utf-8')
+        
+        if requested_hostname is not None:
+            requested_hostname = requested_hostname.decode('utf-8')
```

Comment:
Would be better to skip the following logic flow in case request hostname is none, so we can put a continue statement in else statement for now, later I have some plans for some more refactoring.

Reply:
Added the continue